Select_Flick_Week2/FavoriteMovieDTO.swift

```diff
@@ -9,7 +9,11 @@
 import Foundation
 import RealmSwift
 
-class FavoriteMovieDTO : Object {
-    dynamic var username:String!
-    dynamic var favouriteMovies:[Int:Bool]!
+class FavoriteMovieDTO: Object {
```

Comment:
In .NET mindset, DTO object is an model which coordinate directly with Db. 
I consider Realm is our DBMS, FavoriteMovieDTO is exactly a schema of table. Let me investigate the similar term in Swift. 👯